eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -3,7 +3,7 @@ parameters:
   PreIntegrationSteps: ""
   PostIntegrationSteps: ""
   ResourceServiceDirectory: ""
-  EnvVars: []
+  EnvVars: {}
```

Comment:
Is this required?

Reply:
A valid default for `EnvVars` is required in cases where we don't specify `EnvVars`. In this case we're setting relevant environment variables in the `output` section of the ARM template.